recipes/nn.rb

```diff
@@ -126,6 +110,8 @@
   end
 end
 
+include_recipe "hops::hdfs_user"
```

Comment:
Doing this here is too late. At this time the namenode should have already started. If you see the existing code, the certificate generation is done earlier.

Reply:
Moved.